File: Easy/27_remove_element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int k = nums.size();
        for(int i = 0; i < k; i++){
            if(nums[i] == val) {
                nums[i] = nums[k-1];
                nums.pop_back();
                k--;
                i--;
            }
        }
        return k;
    }
};

File: python/easy/27_remove_element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        slow = len(nums)
        i = 0
        while i < slow:
            if nums[i] == val:
                nums[i] = nums[slow-1]
                slow-=1
                i-=1
            i+=1
        return slow
